ext/src/clicker.jl

```diff
@@ -0,0 +1,75 @@
+function DynamicalSystems.interactive_clicker(ds;
+        # DynamicalSystems kwargs:
+        tfinal = (1000.0, 10.0^4),
+        complete = (x, y) -> [x, y],
+        project = identity,
```

Comment:
Why are these keywords provided here? Shouldn't we expect a `ProjectedDynamicalSystem` directly instead of allowing the projection to happen within the function?